agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -14,6 +14,10 @@
 
 #include <beerocks/tlvf/beerocks_message.h>
 
+#include <tlvf/ieee_1905_1/tlvMacAddress.h>
+#include <tlvf/wfa_map/tlvApMetric.h>
```

Comment:
Hum, apparently the yaml file is not entirely accurate: it is called "AP Metrics TLV" so this should be `tlvApMetrics`.